.github/pull_request_template.md

```diff
@@ -24,5 +24,6 @@ Please add a note if you need help with adding tests.
 -->
 
 - [ ] Checks pass
+- [ ] Update CHANGELOG
```

Comment:
Small nitpick suggestion: use ["changelog" or "change log"](https://en.wikipedia.org/wiki/Changelog). This would better cover alternative file names such as [CHANGES.md](https://github.com/nextstrain/seasonal-flu/blob/3b5126d98396a263302bd1cf7b1c069aef725a7d/CHANGES.md).

```suggestion
- [ ] Update changelog
```